### PR TITLE
MES-4719/test report title

### PR DESCRIPTION
**Pull request by andrewsetterfield**
## Description
Story to conditionally change part of the title on the test report screen.
R/F: if test category matches one of the following EUA1M1, EUA1M2, EUA2M1, EUA2M2, EUAM1, EUAM2, EUAMM1, EUAMM2.
D/F: for all other categories

https://jira.dvsacloud.uk/browse/MES-4719

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [x] One review from each scrum team
- [x] Squashed commit contains the JIRA ticket number

## Screenshots (optional)

Cat EUAM1
Cat B

### CODE REVIEW

File: src/pages/test-report/components/driving-fault-summary/__tests__/driving-fault-summary.spec.ts [hunk 1]
```diff
@@ -71,4 +71,26 @@ describe('DrivingFaultSummary', () => {
...
+      const driverType = component.driverTypeSwitch(TestCategory.EUAM1);
+      expect(driverType).toEqual('R');
+    });
+
+    it('should return R when a category equals EUAM2', () => {
+      const driverType = component.driverTypeSwitch(TestCategory.EUAM2);
+      expect(driverType).toEqual('R');
+    });
+
+    it('should return R when a category equals B', () => {
```
Comment (matthew2564) [inline src/pages/test-report/components/driving-fault-summary/__tests__/driving-fault-summary.spec.ts]: should be - 'should return D when a category equals B'

File: src/pages/test-report/components/driving-fault-summary/__tests__/driving-fault-summary.spec.ts [hunk 2]
```diff
@@ -71,4 +71,26 @@ describe('DrivingFaultSummary', () => {
...
+      const driverType = component.driverTypeSwitch(TestCategory.B);
+      expect(driverType).toEqual('D');
+    });
+
+    it('should return R when a category equals B+E', () => {
```
Comment (matthew2564) [inline src/pages/test-report/components/driving-fault-summary/__tests__/driving-fault-summary.spec.ts]: same as above

File: src/pages/test-report/components/driving-fault-summary/__tests__/driving-fault-summary.spec.ts (hunk 1 above)
Comment (andrewsetterfield) [reply src/pages/test-report/components/driving-fault-summary/__tests__/driving-fault-summary.spec.ts]: corrected

File: src/pages/test-report/components/driving-fault-summary/__tests__/driving-fault-summary.spec.ts (hunk 2 above)
Comment (andrewsetterfield) [reply src/pages/test-report/components/driving-fault-summary/__tests__/driving-fault-summary.spec.ts]: corrected

**Review by MarkPatt89 — APPROVED** [review]

**Review by matthew2564 — APPROVED** [review]

**Pull request merged**